test/jasmine/tests/command_test.js

```diff
@@ -0,0 +1,477 @@
+                var result = Plots.computeAPICommandBindings(gd, 'relayout', [{height: 500, width: 100}]);
+                expect(result).toEqual([{type: 'layout', prop: 'height', value: 500}, {type: 'layout', prop: 'width', value: 100}]);
+            });
+        });
+
+        describe('with astr + val notation', function() {
+            it('and an attribute', function() {
+                var result = Plots.computeAPICommandBindings(gd, 'relayout', ['width', 100]);
+                expect(result).toEqual([{type: 'layout', prop: 'width', value: 100}]);
+            });
+
+            it('and nested atributes', function() {
+                var result = Plots.computeAPICommandBindings(gd, 'relayout', ['margin.l', 10]);
+                expect(result).toEqual([{type: 'layout', prop: 'margin.l', value: 10}]);
+            });
+        });
+
+        describe('with mixed notation', function() {
+            it('containing aob + astr', function() {
+                var result = Plots.computeAPICommandBindings(gd, 'relayout', [{
+                    'width': 100,
+                    'margin.l': 10
+                }]);
+                expect(result).toEqual([
+                    {type: 'layout', prop: 'width', value: 100},
+                    {type: 'layout', prop: 'margin.l', value: 10}
+                ]);
+            });
+        });
+    });
+
+    describe('update', function() {
+        it('computes bindings', function() {
+            var result = Plots.computeAPICommandBindings(gd, 'update', [{
+                y: [[3, 4, 5]],
+                'marker.size': [10, 20, 25],
+                'line.color': 'red',
+                line: {
+                    width: [2, 8]
+                }
+            }, {
+                'margin.l': 50,
+                width: 10
+            }, [1]]);
+
+            expect(result).toEqual([
+                {type: 'data', prop: 'y', traces: [1], value: [[3, 4, 5]]},
+                {type: 'data', prop: 'marker.size', traces: [1], value: [10]},
+                {type: 'data', prop: 'line.color', traces: [1], value: ['red']},
+                {type: 'data', prop: 'line.width', traces: [1], value: [2]},
+                {type: 'layout', prop: 'margin.l', value: 50},
+                {type: 'layout', prop: 'width', value: 10}
+            ]);
+        });
+    });
+
+    describe('animate', function() {
+        it('binds to the frame for a simple animate command', function() {
+            var result = Plots.computeAPICommandBindings(gd, 'animate', [['framename']]);
+
+            expect(result).toEqual([{type: 'layout', prop: '_currentFrame', value: 'framename'}]);
+        });
+
+        it('binds to nothing for a multi-frame animate command', function() {
+            var result = Plots.computeAPICommandBindings(gd, 'animate', [['frame1', 'frame2']]);
+
+            expect(result).toEqual([]);
+        });
+    });
+});
+
+describe('component bindings', function() {
```

Comment:
How is this suite different than the new `binding_test.js` here:

https://github.com/plotly/plotly.js/pull/1016/files#diff-7f09c177b80364e96a43622b17a9d8e7R24

?